### PR TITLE
If skipRedirectOnRemove is enabled, will not redirect to marketplace …

### PR DESCRIPTION
**Pull request by kate1012**
…when instance is disabled [sc-16265]

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "url": "https://github.com/prismatic-io/embedded.git"
   },
   "license": "MIT",
-  "version": "2.3.0",
+  "version": "2.3.1",
```
Comment (jpeters5392) [inline package.json:21]: You should probably update the version in the package-lock too.

File: src/lib/configureInstance.ts [hunk 2]
```diff
@@ -14,8 +14,20 @@ export const configureInstance = ({
...
+        ...options,
+        screenConfiguration: {
+          ...options.screenConfiguration,
+          instance: {
+            ...options.screenConfiguration?.instance,
+            skipRedirectOnRemove: true,
+          },
+        },
+      }
+    : { ...options };
```
Comment (jpeters5392) [inline src/lib/configureInstance.ts:28]: You probably don't need to spread this

File: src/utils/iframe.ts [hunk 3]
```diff
@@ -30,10 +30,11 @@ export const isIframe = (
 ): element is HTMLIFrameElement =>
   Boolean(element && element.tagName === "IFRAME");
 
+type QueryParamKey = "integrationName";
 export const setIframe = (
   route: string,
   options: Options,
-  params?: Record<string, string>
+  params?: Partial<Record<QueryParamKey, string>>
```
Comment (jpeters5392) [inline src/utils/iframe.ts:37]: Is this the only key left in the record now?

Comment (kate1012) [reply src/utils/iframe.ts:37]: Yep! Only using it for integrationName now

Comment (bkegley) [reply src/utils/iframe.ts:37]: Yeah we decided to keep this around even though it's the only thing still using query params. It _kinda_ makes sense to leave that as it's specific to the `find-integration` route.

**Review by bkegley — APPROVED** [review]
Looks good!

**Comment by taylorreece** [comment]
Closing in favor of a different solution.

**Pull request closed without merging**